organization/ddl-scripts/create_tables.cql

```diff
@@ -0,0 +1,96 @@
+-- This CQL script will create the keyspace and all tables needed for the this sample. 
```

Comment:
Please include a link to wherever this code was taken from in a comment or other documentation. Its important to keep references like that easily accessible. Please do the same for any other files adapted for our use from other projects